get_env_data.py

```diff
@@ -21,10 +21,9 @@
     with open(env_file, "w") as txt:
         txt.write("PLEX_TOKEN=-\n")
 
-trakt.APPLICATION_ID = '65370'
-trakt.core.AUTH_METHOD=trakt.core.OAUTH_AUTH
+trakt.core.AUTH_METHOD=trakt.core.DEVICE_AUTH
 trakt_user = input("Please input your Trakt username: ")
-trakt.init(trakt_user, store=True)
+trakt.init(store=True)
```

Comment:
`trakt_user` is no longer used for trakt auth. do we need to keep asking username from the user?

Reply:
You're wrong, it is still used. Read main.py

Reply:
I'm not wrong, it's no longer used for trakt auth in that file. that's why asked is it not used elsewhere as well.